### ISSUE TITLE
Investigate aux-key.e2e-spec.ts failure

### ISSUE THREAD

**Issue by plabadie** [SAP/spartacus#14905]
accessibility/aux-key.e2e-spec.ts is failing on the CI.  It was flagged as flaky.

See #14900

Investigate and fix the issue.

**Comment by plabadie** [comment]
fixed by #14903
